typescript/src/accounts/solana/toSolanaAccount.ts

```diff
@@ -85,6 +94,44 @@ export function toSolanaAccount(
         from: account.address,
       });
     },
+    async quoteFund(options: Omit<SolanaQuoteFundOptions, "address">): Promise<SolanaQuote> {
+      Analytics.trackAction({
+        action: "quote_fund",
+        accountType: "solana",
+        properties: {
+          network: "solana",
+        },
```

Comment:
`network` should be `options.network` (we want to track the user-inputted value of which network they are actioning on!)

Reply:
it's always solana for solana blockchain onramp...

Reply:
maybe this properties field here should be removed altoghether?

Reply:
Oh duh, can't onramp onto a testnet. The value should be `mainnet` then (based on the definition [here](https://github.com/coinbase/cdp-sdk/blob/main/typescript/src/actions/solana/utils.ts#L11)